packages/twenty-front/src/modules/ui/input/components/internal/date/components/InternalDatePicker.tsx

```diff
@@ -220,34 +221,70 @@ const StyledContainer = styled.div`
   }
 `;
 
+const StyledButtonContainer = styled.div`
+  align-items: center;
+  align-self: stretch;
+  display: flex;
+  gap: 8px;
+  height: 32px;
+  padding: 0px var(--Spacing-4px, 4px);
+`;
+
+const StyledButton = styled.button`
+  align-items: center;
+  background-color: transparent;
+  border: none;
+  color: gray;
```

Comment:
Please use the theme for your CSS, here : 

```CSS
color: ${({ theme }) => theme.font.color.secondary};
```

You can find the right color in Figma : 

<img width="273" alt="image" src="https://github.com/twentyhq/twenty/assets/26528466/5f39f591-93ed-4983-b8dd-0884cb1df87d">